readme.md

```diff
@@ -1,5 +1,9 @@
 # C# code generator for AutoRest V3
 
+We have updated the default generator to the new version `--v3`, which uses `@autorest/csharp` package and it will have few side effects:
```

Comment:
I don't think the formatting works 
![image](https://user-images.githubusercontent.com/1697911/107590043-1711d380-6bbc-11eb-8546-304141e42cc7.png)

Reply:
Yeah! just fixed it.